### PR TITLE
[stable/jenkins] Added slave Pod env vars

### PR DESCRIPTION
**Pull request by ecejjar**
Signed-off-by: ecejjar <jesus.javier.arauz@ericsson.com>

#### What this PR does / why we need it:

This PR adds the ability to define environment variables for the Jenkins slave pod.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x ] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by ecejjar** [comment]
Hi guys, anything not right with this PR?

**Comment by ecejjar** [comment]
Guys, can anybody please approve this small but useful PR? Thanks!

File: stable/jenkins/templates/config.yaml [hunk 1]
```diff
@@ -94,7 +94,14 @@ data:
                   </envVars>
                 </org.csanchez.jenkins.plugins.kubernetes.ContainerTemplate>
               </containers>
-              <envVars/>
+              <envVars>
+{{- range $index, $var := .Values.Agent.envVars }}
```
Comment (cpanato) [inline stable/jenkins/templates/config.yaml:104]: it is missing the envVars in the values.yaml

**Review by cpanato — CHANGES_REQUESTED** [review]
see my comment above

File: stable/jenkins/templates/config.yaml (hunk 1 above)
Comment (ecejjar) [reply stable/jenkins/templates/config.yaml:104]: True, I'm adding it

**Comment by maorfr** [comment]
Possible duplicate of helm/charts#8105

**Comment by n1try** [comment]
Any news here? I'd love to have this feature!

**Comment by maorfr** [comment]
/hold cancel

As there seems to be no movement on the duplicate PR, can we get this one going again?

**Pull request merged**